src/detectors/FHCAL/CalorimeterHit_factory_HcalEndcapPRecHits.h

```diff
@@ -25,18 +25,18 @@ class CalorimeterHit_factory_HcalEndcapPRecHits : public eicrecon::JFactoryPodio
         m_input_tag = "HcalEndcapPRawHits";
 
         // digitization settings, must be consistent with digi class
-        m_capADC=1024;//{this, "capacityADC", 8096};
-        m_dyRangeADC=3.6 * dd4hep::GeV;//{this, "dynamicRangeADC", 100. * dd4hep::MeV};
-        m_pedMeanADC=20;//{this, "pedestalMean", 400};
-        m_pedSigmaADC=0.8;//{this, "pedestalSigma", 3.2};
+        m_capADC=4096;//{this, "capacityADC", 8096};
+        m_dyRangeADC=50 * dd4hep::MeV;//{this, "dynamicRangeADC", 100. * dd4hep::MeV};
```

Comment:
50 MeV dynamic range per cell does not make sense to me. Could you clarify?
```suggestion
        m_dyRangeADC=50 * dd4hep::MeV;
```

Reply:
@steinber Can you comment on the 50 MeV dynamic range here? Or @veprbl can you comment on what you think it should be? Trying to wrap this up.

Reply:
This is the old HCal class which will not be run by default in the new simulations. 
But for reference the values should be in `RawCalorimeterHit_factory_HcalEndcapPRawHits.h`: 
`
        m_tRes = 0.0; // in ns
        m_capTime = 100 ; // given in ns, 4 samples in HGCROC
        m_capADC=65536;//2^16
        m_dyRangeADC=1 * dd4hep::GeV;
        m_pedMeanADC=20;
        m_pedSigmaADC = 0.8;
        m_resolutionTDC = 10 * dd4hep::picosecond;
        m_corrMeanScale = 1.0;
`
and in `CalorimeterHit_factory_HcalEndcapPInsertRecHits.h`

`
       m_capADC=65536;//2^16
        m_dyRangeADC=1 * dd4hep::GeV;
        m_pedMeanADC=20;
        m_pedSigmaADC=0.8;
        m_resolutionTDC=10 * dd4hep::picosecond;

        // zero suppression values
        m_thresholdFactor=1.0;
        m_thresholdValue=3.0;

        // energy correction with sampling fraction
        m_sampFrac=0.033;

`

effectively same as for the LFHCal.

Reply:
Can you please make those changes as a commit on this branch? Then we will cherry-pick that commit back into the original branch.

Reply:
Sorry I didn't see the comments before - these were not "per cell" (since the cells were not implemented) but per tile, and I was just trying to get the distribution I was observing in range (for 20 GeV pions) without too much wasted space. That said, i acknowledge it was not fully optimized -- but as Fredi said this is all slated to be fully replaced. I will try and make the changes to close this out today.

Reply:
Cannot cherry-pick into #600 due to permissions, but I assume git will be smart enough to present that as a conflict that needs manual resolution anyway.